### ISSUE TITLE
Broken Library Gensis Engine

### ISSUE THREAD

**Issue by etanot** [searx/searx#1693]
When I searched [linux programming](https://searx.me/?q=%21lg%20linux%20programming&categories=none&language=en-US) keyword with `!lg` prefix, I get this error message 
```
Error! Engines cannot retrieve results.
 library genesis (request exception)
Please, try again later or find another searx instance.
```

I think, it's due to `Library Gensis` is switched to new domain URL (https://libgen.is)<sup>[[1](https://en.wikipedia.org/wiki/Library_Genesis)]</sup> with TLS/SSL support and older URL (http://libgen.io) used in setting.yml<sup>[[2](https://github.com/asciimoo/searx/blob/master/searx/settings.yml#L410)]</sup> file is not reachable and also it doesn't support TLS/SSL. So, we should switch to https://libgen.is URL.

**Comment by alexdelorenzo** [comment]
Can confirm that this error both exists and is fixed by @finn0's suggestion to change the `search_url` in `settings.yml` to point to https://libgen.is/

**Comment by return42** [comment]
If your problem is not solved with the merge of #1694, then please read https://github.com/asciimoo/searx/pull/1694#discussion_r360885319 and https://github.com/asciimoo/searx/pull/1694#discussion_r361150980
